pkgs/by-name/ma/mapserver/package.nix

```
@@ -2,6 +2,9 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  writeScript,
+  common-updater-scripts,
+  coreutils,
```

Alphabetical re-ordering + remove `coreutils` which doesn't look to be used/needed.

```suggestion
  lib,
  stdenv,
  common-updater-scripts,
  fetchFromGitHub,
  writeScript,
```